src/awscli_login/__main__.py

```diff
@@ -141,9 +149,51 @@ def wrapper(args: Namespace, session: Session):
                 exit(code)
 
         return wrapper
+
     return decorator
 
 
+def windowsdaemonize(profile, role, expires):
+    python_path = sys.executable
+    python_path = os.path.abspath(python_path)
+    python_dir = os.path.dirname(python_path)
+    pythonw_path = python_dir + '/pythonw.exe'
+    success_timeout = 30
+    with _LocalNamespacePasser() as worker_argpath:
+        # Write an argvector for the worker to the namespace passer
+        worker_argv = [
+            profile,  # namespace_path
+            role,
+            expires
+        ]
+        with open(worker_argpath, 'wb') as f:
+            # Use the highest available protocol
+            pickle.dump(worker_argv, f, protocol=-1)
+
+        # Create an env for the worker to let it know what to do
+        worker_env = {}
+
+        worker_env.update(dict(os.environ))
+        # Figure out the path to the current file
+        # worker_target = os.path.abspath(__file__)
+        worker_cmd = ('"' + python_path + '" -m ' +
```

Comment:
If this is running on python 3.6 or newer then f-strings might make this more readable. Something like `worker_cmd = f"{python_path} -m awscli_login.windaemon {worker_argpath}"`

Reply:
I'm not sure everyone using this is using > 3.6 python.

Reply:
I went ahead and changed it.